benchmarks/benchmarks/stats.py

```diff
@@ -174,3 +174,24 @@ def time_binned_statistic_dd_reuse_bin(self, statistic):
         stats.binned_statistic_dd(
             [self.inp[0], self.inp[1]], self.inp[2], statistic=statistic,
             binned_statistic_result=self.ret)
+
+
+class ContinuousFitAnalyticalMLEOverride(Benchmark):
+    pretty_name = "Fit Methods Overriden with Analytical MLEs"
+    param_names = ['distribution name']
+    params = ["pareto", "laplace"]
+
+    distributions = {"pareto": {"self": stats.pareto, "shapes": {"b": 2},
+                                "fixed_param": {"floc": 0}},
+                     "laplace": {"self": stats.laplace, "shapes": {},
+                                 "fixed_param": {}}
+                     }
+
+    def setup(self, dist_name):
+        self.distn = self.distributions[dist_name]["self"]
+        self.shapes = self.distributions[dist_name]["shapes"]
+        self.fixed_param = self.distributions[dist_name]["fixed_param"]
+        self.data = self.distn.rvs(size=10000, **self.shapes)
```

Comment:
I'm new to the benchmark code but if it's really only taking 7e4 microseconds = 70ms then let's increase this number 2-5x. for more before and after resolution.

Reply:
I think you are right. It looks like several others have times that take over a second.